src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -345,6 +347,26 @@ def load_command_table(self, _):
         g.command('delete', 'delete')
         g.custom_command('set-primary', 'failover_group_failover')
 
+    ###############################################
+    #             sql instance-pool               #
+    ###############################################
+
+    instance_pools_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#InstancePoolsOperations.{}',
+        client_factory=get_sql_instance_pools_operations)
+    with self.command_group('sql instance-pool', instance_pools_operations, client_factory=get_sql_instance_pools_operations) as g:
```

Comment:
Can you make the new command group with preview for one or two sprints to give us more flexibility to change the command behavior? https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#preview-commands-and-arguments